### ISSUE TITLE
The Given Key was not present error 

### ISSUE THREAD

**Issue by VeissRysto** [hikarin-dev/Auto-Crafter#6]
i was encountering a issue with your trigger. i just want to know if im doing something wrong

![Capture](https://github.com/gynryiie/Auto-Crafter/assets/138848671/5625228f-7eee-477a-a530-6e47c90f9a2d)
![Capture1](https://github.com/gynryiie/Auto-Crafter/assets/138848671/2c907732-a40d-4cb9-b5dd-f955de234b7e)

**Comment by hikarin-dev** [comment]
Try to describe more what your issue is or see https://github.com/gynryiie/Auto-Crafter/issues/3 if that solves your problem, that error is just a harmless bug that I can't get rid of.

**Comment by VeissRysto** [comment]
i was trying to setup the the trigger for the auto crafter and i followed the set up as well for the Dalamud and ACT. also if i hear a beep noise after inputing /e ac , does it mean its working?

**Comment by hikarin-dev** [comment]
yes, the beep should mean the trigger got toggled on or off, you should see a text overlay like this at the top right of your screen if the trigger is on and active
![image](https://github.com/gynryiie/Auto-Crafter/assets/19721540/fd7b3da4-95ff-457e-af8d-f0a0291c72e8)
